PipelinesTasks/PRMetrics/src/overview.md

```diff
@@ -1,5 +1,7 @@
 This extension provides a build task for updating PR titles with an indicator of
-the PR size and test coverage.
+the PR size and test coverage. It works with Azure DevOps, GitHub, GitHub AE,
+and GitHub Enterprise PRs (although functionality is currently limited for
+GitHub platforms).
```

Comment:
nit: is it worth mentioning what the limitation is here?

Reply:
I've updated the details below to highlight the functionality that is currently limited to Azure DevOps.